README.md

```diff
@@ -43,6 +43,18 @@ php occ config:app:set --value="/usr/local/share/pandoc/data/bbcode_phpbb.lua"
 -- pandoc default_output_format
 ```
 
+### PDF support
+
+Pandoc requires a separate PDF engine to be used to convert files to PDF. The
+engine can be configured and will be passed as `--pdf-engine` option to pandoc
+directly.
```

Comment:
How about explicitly mentioning `pdflatex` as a tested an working PDF engine and a short note how to install it?

```suggestion
directly.
```
```suggestion
directly.

Pdflatex (from the texlive project) is one option. On Debian based systems, it can be installed with the `texlive-latex-base` package.
```

Reply:
Good point, adjusted also to the auto detection